packages/dd-trace/src/plugins/ci_plugin.js

```diff
@@ -115,6 +115,18 @@ module.exports = class CiPlugin extends Plugin {
       })
       this.telemetry.count(TELEMETRY_ITR_SKIPPED, { testLevel: 'suite' }, skippedSuites.length)
     })
+
+    this.addSub(`ci:${this.constructor.id}:known-tests`, ({ onDone }) => {
```

Comment:
Not blocking, but it's a bit difficult to know why this is needed without any publisher. Generally speaking it's a pattern that should be avoided, so just pointing it out to make sure other options were evaluated.

Reply:
that's a good point. The changes are considerable so I split the PRs, leaving this is as a noop, which isn't great. The usage will become clear very soon though, as I have the changes ready :)